src/chain-data/state.ts

```diff
@@ -118,6 +118,34 @@ interface Vesting {
   remainingToWithdraw: BigNumber;
 }
 
+export interface Claim {
+  claimId: string;
+  policyId: string;
+  evidence: string;
+  timestamp: Date;
+  claimant: string;
+  beneficiary: string;
+  claimedAmount: number;
```

Comment:
Yep I saw that 👍 (I used BigNumber in the mock contract data):
```ts
const mockContractData = [
  {
    claimId: BigNumber.from(2),
    ...
    timestamp: BigNumber.from(1652191585),
    claimedAmount: BigNumber.from(200),
    counterOfferAmount: null,
    resolvedAmount: BigNumber.from(200),
    status: 6,
    statusUpdatedAt: BigNumber.from(1652191585),
  },
```
 Wouldn't we want to convert them once upfront though, instead of using `.toNumber()` in all the different places where we use it? (similar to how we convert these dates to Date objects)

Reply:
It's better to do the math operations using the BigNumber methods. When displaying the numbers in the UI use one of the formatting helper methods (that inserts commas and rounds to certain number of decimals places).

Reply:
Ah ok gotcha sounds good 👍